### ISSUE TITLE
I keep hitting NaN value error.

### ISSUE THREAD

**Issue by FeepingCreature** [manicman1999/StyleGAN2-Tensorflow-2.0#4]
After a few minutes of training.

**Comment by IsaacEthington** [comment]
I experienced the same thing. I kept getting NAN errors after just a few minutes. The learning rate may be too large. Did you adjust the learning rate? Try setting it lower. I haven't had any problems since then.
